mypyc/genops.py

```diff
@@ -1975,11 +1977,44 @@ def decompose_union_helper(self,
         self.activate_block(exit_block)
         return result
 
-    def isinstance(self, obj: Value, rtype: RInstance, line: int) -> Value:
-        class_ir = rtype.class_ir
-        fullname = '%s.%s' % (class_ir.module_name, class_ir.name)
-        type_obj = self.load_native_type_object(fullname)
-        return self.primitive_op(fast_isinstance_op, [obj, type_obj], line)
+    def isinstance_helper(self, obj: Value, class_irs: List[ClassIR], line: int) -> Value:
+        """Fast path for isinstance() with native classes."""
```

Comment:
This docstring was a bit confusing at first. Maybe something like "... that checks against a list of native classes."

Reply:
OK.